### ISSUE TITLE
Create option to modify the manifest

### ISSUE THREAD

**Issue by halatmit** [mit-cml/appinventor-sources#795]
Provide a way for advanced users to modify the manifest before building.
The was suggested by Taifun in response to a user query about adding uses-sdk 
to the manifest:
https://groups.google.com/forum/#!topic/mitappinventortest/vJht0JipC_g
I'm mixed on doing this, because app-to-market is available, but maybe we can provide it
if we can do it in such a way that it does not bother/confuse the majority of users.

**Comment by weihuali0509** [comment]
I have to warn you - if you put the target sdk to be =>21, a lot of app inventor components will be broken because of the material design theme addition.

**Comment by ewpatton** [comment]
I don't think we need to provide a general way of editing the whole manifest. Instead, we should identify key pain points and provide options in the project properties to control for those. For example, I see a lot of people wanting to bump their minSdk, and so having this as a property rather than letting people edit the raw manifest XML seems more App Inventor-y. Another thing is sometimes people want to remove permissions that are not really needed but added due to the conservative assumptions of some components. An editor where they could indicate what permissions are used (and maybe what components/extensions add them) could also be useful. There may be others so I've marked this as needs discussion.
